src/web/components/Caption.tsx

```diff
@@ -8,11 +9,18 @@ import TriangleIcon from '@frontend/static/icons/triangle.svg';
 const figureStyle = css`
     margin-bottom: 8px;
 `;
-const captionStyle = css`
+const captionStyle = (role?: RoleType) => css`
     padding-top: 10px;
     ${textSans.xsmall()};
     word-wrap: break-word;
     color: ${palette.neutral[46]};
+    ${from.leftCol} {
+        width: ${role && (role === 'showcase' || role === 'supporting') && '140px'};
```

Comment:
All these CSS lines (18–19 + 22) have the `role && (role === 'showcase' || role === 'supporting')` pattern. Would it be easier to read if we wrap the whole thing with that condition? And, if so, can you please show me how I'd write that?

Reply:
Thanks @duarte , this is amazing 🎉 

I've added a commit to solve the repeating logic problem. I just created a new css class and then set a conditional for when it is applied. Check it over your end to make sure I didn't miss any use cases.

I also ran Prettier on the file to keep out CI server happy. For this project it helps to have it set so it runs Prettier automatically on save (umm, assuming you're using VSCode) to avoid annoying linting errors. You can also run `make validate` and it will check locally for these things.